Project.toml

```diff
@@ -19,7 +19,7 @@ IntervalArithmetic = "^0.20"
 IntervalRootFinding = "0.5"
 RecipesBase = "1"
 Reexport = "1"
-TaylorIntegration = "0.9"
+TaylorIntegration = "0.9, 0.10"
 TaylorSeries = "0.12"
 julia = "1.6, 1.7"
```

Comment:
Thanks @lucaferranti for solving this. What about also increasing the julia version to include 1.8?

Reply:
AFAIK, once the major version is bigger than 0, you only have to list the supported major versions. So the 1.7 is already redundant. The next version you would have to add is 2.

Reply:
Yep, exactly

Reply:
Having `julia = "1.6"` means [1.6, 2.0)

Reply:
Thanks @schillic for the clarification! So, let's go ahead and merge this and register the new version!